.vscode-template/launch.json

```diff
@@ -0,0 +1,18 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Python: Remote Attach storage",
+      "type": "python",
+      "request": "attach",
+      "port": 3000,
+      "host": "127.0.0.1",
+      "pathMappings": [
+        {
+          "localRoot": "${workspaceFolder}/services/storage",
+          "remoteRoot": "/devel/services/storage"
+        }
```

Comment:
So this is ONLY for development mode, right?

Reply:
this example yes